core/runtime/jsruntime.rs

```diff
@@ -922,6 +922,11 @@ impl JsRuntime {
       }
     }
 
+    state_rc
+      .source_mapper
+      .borrow_mut()
+      .set_module_map(module_map.clone());
```

Comment:
This needs to be rearchitected. I'm pretty sure some test would fail, because there's now one additional reference to `ModuleMap`.